build.gradle

```diff
@@ -230,12 +232,12 @@ dependencies {
 
     implementation 'javax.xml.bind:jaxb-api:2.3.1'
 
-    // Google Protocol Buffers are used in decoding OSM data and seamless census data.
-    implementation 'com.google.protobuf:protobuf-java:2.6.1'
+    // Google Protocol Buffers are used in decoding OSM data and seamless census data (in Geobuf format).
+    // implementation 'com.google.protobuf:protobuf-java:3.13.0'
 
-    // OSM PBF format support. Transitive dependency on Protobuf 3.4.0.
-    // TODO investigate any conflict with above
-    implementation 'org.openstreetmap.osmosis:osmosis-osm-binary:0.45'
+    // OSM PBF format support. Transitive dependency on Protobuf 3.12.2,
+    // align with mapbox-vecor-tile and geobuf generated classes
+    implementation 'org.openstreetmap.osmosis:osmosis-osm-binary:0.48.3'
```

Comment:
To summarize: seamless census was depending on 2.6.1 via this import and osmosis was pulling in 3.4.0; now the single 3.12.2 version (compatible with mapbox-vector-tile and geobuf) is used via transitive dependency from osmosis.